docs/usage-cli.md

```diff
@@ -102,7 +102,8 @@ Extra header to include in every request. This option can be used multiple times
 Show usage information.<br>
 
 ### --hookfiles, -f
-Specifies a pattern to match files with before/after hooks for running tests<br>
+Specifies a pattern to match files with before/after hooks for running tests. Files are executed in
```

Comment:
Addressed in https://github.com/apiaryio/dredd/pull/803/commits/e867e4ad0e11b95ff53cb867c5ea3c5943d7e6e3.